tests/acceptance/10_files/copy_local_file.cf

```diff
@@ -0,0 +1,48 @@
+bundle common test_meta
+{
+  vars:
+      "description" string => "Test that copying a file locally works";
+      "story_id" string => "5576";
+      "covers" string => "copy_local_file";
```

Comment:
covers should be "operational_repaired" here (but this string is still unused).

operational = not dry-run
repaired = you're testing that something not in compliance became in compliance